app/ota_client.py

```diff
@@ -565,15 +566,16 @@ def _download_raw_file(self, url, dest_file, target_hash="", fl=""):
             logger.error(f"hash missmatch: {dest_file}")
             logger.error(f"  dl hash: {digest}")
             logger.error(f"  hash: {target_hash}")
+            fl = global_var_dict["tmp-queue-log_queue"]
             if fl != "":
-                fl.write("hash missmatch: " + dest_file + "\n")
+                fl.put("hash missmatch: " + dest_file + "\n")
```

Comment:
This log file is not necessary. So remove this lines, and also remove log file server process.

Reply:
Resolve in commit 90f2727.